docs/src/_guide/plugin-management.md

```diff
@@ -480,6 +480,9 @@ If you've forked a plugin's repository and made changes to it, you can update yo
       pip_url: git+ssh://git@gitlab.com/meltano/tap-gitlab.git
       ```
 
+      Depending on your git provider (such as Azure Repos), some `git+ssh` URLs may contain a colon. This colon can cause errors with `pip`. In this case, try replacing the colon with a forwardslash.
```

Comment:
```suggestion
      Depending on your git provider (such as Azure Repos), some `git+ssh` URLs may contain a colon. This colon can cause errors with `pip`. In this case this can be fixed by replacing the colon with a forward slash.
```